main.go

```diff
@@ -626,6 +648,12 @@ func parseFlags() (config, error) {
 		"The log filtering level. Options: 'error', 'warn', 'info', 'debug'.")
 	flag.StringVar(&cfg.logFormat, "log.format", logger.LogFormatLogfmt,
 		"The log format to use. Options: 'logfmt', 'json'.")
+	flag.StringVar(&cfg.internalTracing.serviceName, "internal.tracing.service-name", "observatorium_api",
+		"The service name to report to the tracing backend.")
+	flag.StringVar(&cfg.internalTracing.endpoint, "internal.tracing.endpoint", "",
+		"The full URL of the trace collector. If it's not set, tracing will be disabled.")
+	flag.Float64Var(&cfg.internalTracing.samplingFraction, "internal.tracing.sampling-fraction", 0.1,
```

Comment:
Is `0.1` a good default for the fraction of traces sampled?

Reply:
Does not matter much, TBH, 0.1 can be IMO